numpy/f2py/crackfortran.py

```diff
@@ -1540,8 +1539,6 @@ def markinnerspaces(line):
             cc = c
         if c == cc:
             f = f + 1
-        elif c == cc:
```

Comment:
This looks like a bug and probably should be simply deleted.

Reply:
I'm wary of changing this code without understanding what was actually intended here. Obviously the current code is nonsense, but cleaning up nonsense while preserving behavior makes it harder to work out whether that behavior is what the nonsense was intended to do in the first place.

Does the git blame reveal the commit that introduced this duplication?

Reply:
> I'm wary of changing this code without understanding what was actually intended here. Obviously the current code is nonsense, but cleaning up nonsense while preserving behavior makes it harder to work out whether that behavior is what the nonsense was intended to do in the first place.
> 
> Does the git blame reveal the commit that introduced this duplication?

I fully agree.  I somehow short circuited the **not** in my op.  It definitely should be investigated.